grpcutil/status.go

```diff
@@ -60,6 +61,17 @@ func ErrorToStatusCode(err error) codes.Code {
 	return codes.Unknown
 }
 
+func Status(errCode codes.Code, errMessage string, details ...proto.Message) *status.Status {
+	stat := status.New(errCode, errMessage)
+	if details != nil {
```

Comment:
Less confusing/fragile:
```suggestion
	if len(details) {
```